### DISCUSSION TITLE
Can advise when can purge content of temp folder

### DISCUSSION THREAD

**Discussion by alexissinglaire** [bpatrik/pigallery2#733] [Q&A]
Hi guys,
I need advise when will it be the good moment to purge the content of temp folder?

Thanks.

**Comment by bpatrik** [comment]
Hi,

There is a job to do that selectively. If you delete the whole folder the
app will need to recreate all thumbnails.

-- Sorry for being brief, sent from my phone.

**Comment by alexissinglaire** [comment]
Hi,

You mean I need to create cron job that selectively choose which folders or
files that need to be purge?

Regards,

**Reply by bpatrik** [reply]
No I mean this: ![Screenshot_20231006-115818.png](https://github.com/bpatrik/pigallery2/assets/7512182/42ca8e0e-5d59-411f-8c95-c54611940edb)

That will delete the unused files

**Reply by bpatrik** [reply]
You need to switch to advanced mode to see it in the settings

**Comment by alexissinglaire** [comment]
Noted. Thanks so much
